src/exchanges.js

```diff
@@ -0,0 +1,75 @@
+let base = require('taskcluster-base');
+
+let exchanges = new base.Exchanges({
+  title: 'S3 Distribute Pulse Exchanges',
+  description: [
+    'Exchanges from the s3 distribution manager',
+  ].join('\n'),
+  schemaPrefix: 'http://schemas.taskcluster.net/aws-provisioner/v1/',
+});
+
+module.exports = exchanges;
+
+let commonRoutingKey = [
+  {
+    // Let's keep the "primary." prefix, so we can support custom routing keys
+    // in the future, I don't see a need for it here. But it's nice to have the
+    // option of adding it later...
+    name: 'routingKeyKind',
+    summary: 'Identifier for the routing-key kind. This is ' +
+             'always `\'primary\'` for the formalized routing key.',
+    constant: 'primary',
+    required: true,
+  }, {
+    name: 'service',
```

Comment:
hmm... this is not fully through through... but perhaps `cloud-mirror` should consider service and region the same thing. I mean `service/region` gives you the backend... So why not use `s3:us-west-2` as `backendName` and then not have concepts of `service` and `region`, but only deal with the concept of `backendName` (which could just be called `backendId` or even better probably `location`).

Sorry, that got messy, suggestion: remove `service` and `region` and just have `location` which can take the form `s3:us-west-2`, `s3:us-east-1`, and when/if we ever do google cloud storage, we can do: `gcs:<region>` for that service.

After `cloud-mirror` only deals with backends... why should it need to know the service, if we can make that a generic concept.